Library/Formula/corectl.rb

```diff
@@ -28,6 +27,11 @@ def install
     args = []
     args << "VERSION=#{version}" if build.stable?
 
+    if build.head?
+      touch "unreleased"
+      system "git", "add", "unreleased"
+    end
```

Comment:
"cosmetics". i wanted homebrew blob built from HEAD to show (via `corectl version` exact same behavior as local git builds - explicitly telling people that they are running something that isn't a tagged version (by having `-unreleased` appended to version. In the Makefile i have `VERSION := $(shell git describe --abbrev=6 --dirty=-unreleased --always --tags)` but as that wasn't enough ence the above snippet.